roles/model/templates/model.deployment.yaml.j2

```diff
@@ -214,6 +221,18 @@ spec:
             - name: mnt
               mountPath: /etc/wisdom/uwsgi.ini
               subPath: uwsgi_ini
+{% if bundle_ca_crt %}
+          - name: "ca-trust-extracted"
```

Comment:
Nit pick, but these could be indented 2 more spaces to be consistent with the other `volumeMounts`. 

(Although in reality the indentation for `mnt` and `server-tls-certs` is 2 spaces too much).